assets/model_monitoring/components/src/action_analyzer/action_analyzer_correlation_test/run.py

```diff
@@ -128,9 +127,6 @@ def get_unique_group_and_index(data_with_action_metric_score_df):
 
 def run():
     """Correlation test."""
-    # setup momo environment
-    init_momo_component_environment()
```

Comment:
Yes we only needed it on create manifest and feature importance component because we don't use StoreUrl in those components. I will switch over to use StoreUrl to write the mltable yml but that will be in a separate PR.